runtime/doc/treesitter.txt

```diff
@@ -101,6 +103,9 @@ TSNode:named_child({index})                              *TSNode:named_child()*
     Get the node's named child at the given {index}, where zero represents the
     first named child.
 
+TSNode:child_containing_descendant({descendant})  *TSNode:child_containing_descendant()*
+    Get the node's child that contains {descendant}.
```

Comment:
does it only return immediate children? if a grandchild has `descendant` will it return the grandchild?

Reply:
Yes, the method only returns immediate children.

I think tree-sitter follows the convention where direct children are called "children", and direct+indirect children are called "descendants".

Should I add "immediate child" into the description?

Reply:
I think we use the same terminology (cf ancestor v. parent)

Reply:
yeah, just checking